app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def oauth_login_path
```

Comment:
Az omniauth-os path-okat nem írja ki a `rake routes`. Elvileg nincs is saját helperjük, SO-n is ezt a módszert ajánlják.